centos-ci/heketi-functional/gluster_heketi-functional.xml

```diff
@@ -21,7 +21,7 @@
         <hudson.model.StringParameterDefinition>
           <name>TEST_SCRIPT</name>
           <description>Test script to execute on the reserved machine.</description>
-          <defaultValue>https://raw.githubusercontent.com/gluster/glusterfs-patch-acceptance-tests/master/centos-ci/heketi-functional/run-test.sh</defaultValue>
+          <defaultValue>https://raw.githubusercontent.com/heketi/ci-tests/centos-ci/heketi-centos-ci-tests.sh</defaultValue>
```

Comment:
having branch name after the repo name "ci-tests" like it was in the original line is preferable. This would allow us to have different test-run script for different branches/jobs.

Reply:
@raghavendra-talur, I am not sure I understand what you mean: The new line has the repo name "ci-tests" followed by the branch name "centos-ci". I redundantly renamed the script to include heketi and centos-ci in the name though. Could you elaborate on your request?

Reply:
Oh, do you mean the **heketi** branch name? That can easily be done. But the original code in this repo also does not use any branches other than master. So this master here does not reflect the heketi branch. Also my current thinking was that it would be good to always use the latest test script for whatever branch. If we wanted a heketi-branch-specific test script, we could as well put them directly into the corresponding heketi branch.

Reply:
@obnoxxx My comment is wrong. I did not see that centos-ci is a branch in heketi/ci-tests repo. Patch looks good to me.